packages/block-editor/src/components/block-list/index.js

```diff
@@ -233,7 +233,6 @@ function Items( {
 						<ZoomOutSeparator
 							clientId={ clientId }
 							rootClientId={ rootClientId }
-							position="top"
```

Comment:
Just trying to understand the changes: why the "position" is not needed anymore?

Reply:
We're not doing anything with it. Unless there is a problem and we need to bring it back. In this PR _both_ separators show up, irrelevant if of top/bottom.